docs/app/Examples/elements/Segment/Variations/SegmentAttachedComplexExample.js

```diff
@@ -5,25 +5,25 @@ export default class SegmentAttachedComplexExample extends Component {
   render() {
     return (
       <div>
-        <Header.H5 className='attached'>
+        <Header.H5 attached>
           Dogs
         </Header.H5>
-        <Segment className='attached segment'>
+        <Segment attached>
           Dogs are one type of animal
         </Segment>
-        <Header.H5 className='attached header'>
+        <Header.H5 attached>
           Cats
         </Header.H5>
-        <Segment className='attached segment'>
+        <Segment attached>
           Cats are thought of as being related to dogs, but only humans think this.
         </Segment>
-        <Header.H5 className='attached header'>
+        <Header.H5 attached>
           Lions
         </Header.H5>
-        <Segment className='attached segment'>
+        <Segment attached>
           Humans don't think of lions as being like cats, but they are.
         </Segment>
-        <Message className='bottom attached warning'>
+        <Message attached='bottom' className='warning'>
```

Comment:
`<Message />` has not yet been updated to v1 API, so the `attached` prop won't work here.